### PR TITLE
[UNR-5424]Work around fix for spatial debugger crashing after client travelling

### PR DESCRIPTION
**Pull request by simonsarginson**

### CODE REVIEW

File: SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp [hunk 1]
```diff
@@ -434,6 +434,12 @@ void ASpatialDebugger::DrawTag(UCanvas* Canvas, const FVector2D& ScreenLocation,
 
 	check(NetDriver != nullptr && !NetDriver->IsServer());
 
+	// TODO: fix this hack for fixing spatial debugger crash after client travel UNR-5481
```
Comment (joshuahuburn) [inline SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp:437]:
```suggestion
	// TODO: UNR-5481 - Fix this hack for fixing spatial debugger crash after client travel
```

File: SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h [hunk 2]
```diff
@@ -95,6 +95,8 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public SpatialOS
 
 	const SpatialGDK::EntityView& GetView() const;
 	SpatialGDK::ViewCoordinator& GetCoordinator() const;
+	// TODO: fix this hack for fixing spatial debugger crash after client travel UNR-5481
```
Comment (joshuahuburn) [inline SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h:98]:
```suggestion
	// TODO: UNR-5481 - Fix this hack for fixing spatial debugger crash after client travel
```

File: SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp [hunk 3]
```diff
@@ -434,6 +434,12 @@ void ASpatialDebugger::DrawTag(UCanvas* Canvas, const FVector2D& ScreenLocation,
 
 	check(NetDriver != nullptr && !NetDriver->IsServer());
```
[the remark this answers is not in the document]
Comment (simonsarginson) [reply SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp:435]: It hasn't happened as far as I know 🤷

[the remark this answers is not in the document]
Comment (simonsarginson) [reply SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp:435]: Certainly for this bug NetDriver and Connection is still valid at this point.

**Review by samiwh — APPROVED** [review]

**Review by joshuahuburn — APPROVED** [review]

**Pull request merged**